contracts/infrastructure/PlatformVoter.sol

```diff
@@ -137,7 +137,7 @@ contract PlatformVoter is ControllableV3, IPlatformVoter {
     _vote(tokenId, _type, value, target, false);
   }
 
-  function _vote(uint tokenId, AttributeType _type, uint value, address target, bool skipDelay) internal {
+  function _vote(uint tokenId, AttributeType _type, uint value, address target, bool onlyRefresh) internal {
```

Comment:
Why the name of the skipDelay was changed? I would suggest to add comment to param with clear description of its purpose, i.e.:
/// @param onlyRefresh Skip delay-restriction
Timestamps of exist votes are not changed below, vote weights are just refreshed now. But refreshing happens without any dependence on the value of onlyRefresh.